spec/lib/saml/ssoe_user_spec.rb

```diff
@@ -359,8 +409,42 @@
           mhv_correlation_id: nil,
           uuid: '1655c16aa0784dbe973814c95bd69177',
           email: 'Test0206@gmail.com',
+          idme_uuid: '1655c16aa0784dbe973814c95bd69177',
+          loa: { current: 3, highest: 3 },
+          sign_in: { service_name: 'dslogon', account_type: 3 },
+          sec_id: '0000028007',
+          multifactor: multifactor,
+          authenticated_by_ssoe: true
+        )
+      end
+    end
+
+    context 'DSLogon premium user with idme uuid in gcIds' do
+      let(:authn_context) { 'dslogon' }
+      let(:account_type) { '3' }
+      let(:highest_attained_loa) { '3' }
+      let(:multifactor) { true }
+      let(:saml_attributes) { build(:ssoe_idme_dslogon_level2, va_eauth_uid: ['0000028007'], va_eauth_csid: ['NOT_FOUND']) }
```

Comment:
@patrickvinograd I added this test case to trigger the new gcId parsing logic in ssoe.rb, however looking at all the new saml_attribute factories you've created it doesn't appear to be possible to get a response back from SSOe with a `va_eauth_csid != 'idme' value.  Is this because we haven't captured any SAML payloads for inbound authentication yet?

Reply:
That's correct. I'll push a branch with those payloads tonight at least for visibility; if you want to cherry pick it into your PR that would work too.